src/Models/LayerModel.vala

```diff
@@ -22,11 +22,7 @@
 public class Akira.Models.LayerModel : Models.ItemModel {
     public string name {
         owned get {
-            if (item.name != null) {
-                return item.name;
-            }
-
-            return item.id;
+            return item.name != null ? item.name : item.id;
```

Comment:
I was thinking to change how we give ID's to item, stick to a pure numeric format and marshall the name (when not set by the user) by getting the `default_layer_name` property of each Canvas Item. This way we could make use of incremental numeric ID and have the complete human readable name of the Layer (like it was on the video of the previous version of the LayerPanel) like "Rectangle 0", "Ellipse 0", "Text 0", rather then the capitalized version of the ID which is present today. What do yo think?